src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/__tests__/TargetingRulesTab.test.tsx

```diff
@@ -64,6 +58,43 @@ describe('TargetingRulesTab', () => {
       loading: false,
       refetch: jest.fn()
     } as any)
+
+    jest.clearAllMocks()
+  })
+
+  test('it should disable this section the selected flag is archived', async () => {
+    const archivedMockFeature = cloneDeep(mockFeature)
+    archivedMockFeature.archived = true
+
+    renderComponent({
+      featureFlagData: {
+        ...archivedMockFeature,
+        envProperties: {
+          pipelineConfigured: false,
+          pipelineDetails: undefined,
+          defaultServe: { variation: 'false' },
+          environment: 'qatest',
+          modifiedAt: 1635333973373,
+          offVariation: 'false',
+          rules: [],
+          state: 'off',
+          variationMap: [],
+          version: 56
+        }
+      }
+    })
+
+    expect(screen.getByText('CF.SHARED.ARCHIVED')).toBeInTheDocument()
+
+    // flag variations dropdowns
+    expect(screen.getAllByRole('textbox')[0]).toBeDisabled()
+    expect(screen.getAllByRole('textbox')[1]).toBeDisabled()
```

Comment:
Almost certainly not a problem. But just for speed of tests, it's usually better to get a variable or a loop going.
```suggestion
    const textboxes = screen.getAllByRole('textbox')
    expect(textboxes[0]).toBeDisabled()
    expect(textboxes[1]).toBeDisabled()
```

Or
```suggestion
    screen.getAllByRole('textbox').forEach(textbox => expect(textbox).toBeDisabled())
```